docs/developers-guide-drivers.md

```diff
@@ -0,0 +1,31 @@
+# Metabase Community Drivers
+
+As of Metabase 0.32, it's now possible to develop your own 3rd-party database drivers! Always wanted Metabase to be able to query the latest and greatest data source? Well now it's possible and all you have to do is write a little Clojure!
+
+There are already a number of existing drivers out there. While we don't specifically endorse any of these drivers, we thought it would be helpful to have a central place to list them.
+
+## Community Database Drivers
+
```

Comment:
I feel like this should be somewhere on www.metabase.com

This is pretty buried. For drivers that we think are high quality we should drive people towards them.

Reply:
Hm I'd say under maybe the "Open source and constantly improving" or "Works with" section in that case? 

One question, though, is what we define as high quality. @camsaul had some thoughts about this in his original PR #9087 (See `A Metabase plugins store`).

I think at the moment "high quality" is equivalent to "updated frequently" and some sort of validation that the driver works as intended.

Reply:
If we wanted to dedicate the time to it we could make it easy to run tests against 3rd-party drivers or set up something like a Metabase plugins store. Since the plugins are loaded dynamically it would be entirely possible to download them and load them up without restarting the instance. So that's something to consider in the future. It's just a matter of prioritizing features, and it would take a while to implement either one

Reply:
If these drivers go on the homepage, I think the "works with" section is the right place. 

I think I'd just list the names of the drivers in the main list, like "Athena." Each of the official drivers could possibly link to the doc page for how to connect to that DB, and all of the high quality community drivers that are listed could all go to the same interstitial page talking about community drivers, with links to corresponding repo pages. I.e., the "works with" section should IMO be more focused on answering the user question "can I use Metabase with the data I have?" rather than doing work to explain the distinction between drivers we've built vs. the community has built — that can be clarified as needed after a click.

Reply:
Cool, sounds good @mazameli 

And @camsaul I agree with you on that it would be nice to have a base set of tests we can run against 3rd-party drivers to validate different sets of functionality. I can think about how to structure that a bit more as I just went through adding CI testing to the Athena driver.

Reply:
what was in my head with "High Quality"  was
1) Full support for the driver API (i.e. some community drivers have only supported native queries)
2) Passes all driver tests
3) Is actively being maintained by someone who commits to keeping them mostly caught up to the latest version.

Reply:
Sounds good - will need to figure out a way to validate #1 and #2 but #3 is pretty easy to keep an eye on. And one of the reasons I added "last release date" to the table. Will rework this a little bit.

Reply:
OK - I added a PR for the website that adds a few drivers that are actively developed and links back to this documentation. I'll open a feature request to make testing/validation easier for third-party drivers.

I also tweaked this documentation a little bit so it's a little bit more friendly for somebody clicking on the home page and wondering "hey do I use that thing?".

Reply:
Added my thoughts for tests to #9348